perpendicular-flap/solid-nutils/requirements.txt

```
@@ -0,0 +1,2 @@
+nutils
```

```suggestion
nutils==8
```
There will be a Nutils 9 :grin:

---

Will this tutorial then also work with version 9? ;)

We can also write `nutils>=8` here, if this is correct.

---

Normally, some things break. So `==8` is what we want.